### DISCUSSION TITLE
Can I join you guys to work on this project?

### DISCUSSION THREAD

**Discussion by zhangjinshui-nerveee** [upb-lea/transistordatabase#75] [Q&A] [answered]

**Comment by gituser789** [comment, marked as answer]
We are looking forward to a possible cooperation. Work your way into the topic and start with simple issues, open a pull request when committing changes. You will get our support, contact us for questions.

**Reply by tinix84** [reply]
Can you list the hot features?

**Reply by gituser789** [reply]
Hi tinix84,
please find the overview here, i also added this to the readme-file. 

```
The key feature is the Python interface, to use the transistor data in self-written optimization routines.

1. Python interface
Use the transistor data in you self-written optimization program, see figure:

* Automatic calculate your converter losses with many different transistors
* Search the database for usable transistors for your application
* Functions provided, to search for closest operating point
* Functions provided, to linearize the transistor/diode conduction behaviour
* Functions provided, to calculate the output capacitances Energy from the C_oss-curve 
* Use own loss measurements for above features (coming soon)

2. GUI

* Manage, store and search the transistors
* Export transistor models to programs like GeckoCIRCUITS, PLECS, Matlab/Octave
* Compare transistors (interpolate switch loss data for new gate resistors and temperatures, ...)
```

**Reply by gituser789** [reply]
There are still some key features missing, like
 * Improve interpolation for all exporters https://github.com/upb-lea/transistordatabase/issues/71
 * Loss correction for resonant topologies, https://github.com/upb-lea/transistordatabase/issues/26
 * Store diode data in the database, https://github.com/upb-lea/transistordatabase/issues/18
